Wikipedia/Code/DescriptionEditViewController.storyboard

```diff
@@ -0,0 +1,248 @@
+                                                                        <rect key="frame" x="0.0" y="0.0" width="258" height="16"/>
+                                                                        <fontDescription key="fontDescription" style="UICTFontTextStyleFootnote"/>
+                                                                        <nil key="textColor"/>
+                                                                        <nil key="highlightedColor"/>
+                                                                    </label>
+                                                                    <label opaque="NO" contentMode="left" horizontalHuggingPriority="251" verticalHuggingPriority="251" text="Login" textAlignment="natural" lineBreakMode="wordWrap" numberOfLines="0" baselineAdjustment="alignBaselines" adjustsFontSizeToFit="NO" translatesAutoresizingMaskIntoConstraints="NO" id="ajM-UL-huc">
+                                                                        <rect key="frame" x="0.0" y="24" width="258" height="16"/>
+                                                                        <gestureRecognizers/>
+                                                                        <fontDescription key="fontDescription" style="UICTFontTextStyleFootnote"/>
+                                                                        <nil key="textColor"/>
+                                                                        <nil key="highlightedColor"/>
+                                                                        <connections>
+                                                                            <outletCollection property="gestureRecognizers" destination="5TN-tg-JtF" appends="YES" id="wCP-HV-LJI"/>
+                                                                        </connections>
+                                                                    </label>
+                                                                </subviews>
+                                                            </stackView>
+                                                        </subviews>
+                                                        <gestureRecognizers/>
+                                                        <connections>
+                                                            <outletCollection property="gestureRecognizers" destination="ZDL-12-khV" appends="YES" id="agN-42-wYi"/>
+                                                        </connections>
+                                                    </stackView>
+                                                </subviews>
+                                                <constraints>
+                                                    <constraint firstAttribute="width" constant="288" id="Ka3-zi-uhG"/>
```

Comment:
The width of the edit content on landscape should be tied to readable width instead of a fixed width. You can make the superview margins follow readable width and constrain the stack view to the margins.

Reply:
fixed